packages/@expo/cli/src/start/server/middleware/InterstitialPageMiddleware.ts

```diff
@@ -20,26 +20,23 @@ const debug = require('debug')(
 
 export const LoadingEndpoint = '/_expo/loading';
 
-function getRuntimeVersion(exp: ExpoConfig, platform: 'android' | 'ios' | null): string {
-  if (!platform) {
-    return 'Undetected';
-  }
-
-  return getRuntimeVersionNullable(exp, platform) ?? 'Undetected';
-}
-
 export class InterstitialPageMiddleware extends ExpoMiddleware {
-  constructor(projectRoot: string) {
+  private scheme: string | null;
+
+  constructor(projectRoot: string, scheme?: string | null) {
```

Comment:
You should change the format, but if this format were to stay, we'd prefer the syntax:
```suggestion
  constructor(projectRoot: string, public scheme: string | null = null) {
```